pilot/pkg/model/push_context.go

```diff
@@ -979,9 +985,10 @@ func (ps *PushContext) initAuthnPolicies(env *Environment) error {
 	return nil
 }
 
-func (ps *PushContext) addAuthnPolicy(hostname host.Name, selector *authn.PortSelector, policy *authn.Policy) {
+func (ps *PushContext) addAuthnPolicy(hostname host.Name, selector *authn.PortSelector, policy *authn.Policy, configMeta *ConfigMeta) {
```

Comment:
So the returned value basically is a `model.Config` except the spec has been type converted to `authn.Policy`, right?

I just think less number of return value the better design . And the ConfigStore's return value is `Config`.

Reply:
I'd like to make the least change at this point. There are a certain benefit of holding policy directly as well, given some modification has been made on it.

Reply:
Sg.